src/main/java/org/mqttbee/api/mqtt/MqttClientExecutorConfig.java

```diff
@@ -42,6 +42,6 @@
     Optional<Integer> getUserDefinedNettyThreads();
 
     @NotNull
-    Scheduler getRxJavaScheduler();
+    Scheduler getApiScheduler();
```

Comment:
That's the scheduler used to execute observers that are attached by the application. Right?

Could we find a better name here. What about ApplicationScheduler or UserScheduler...

Reply:
I will choose ApplicationScheduler